build.gradle

```diff
@@ -79,7 +79,7 @@ allprojects {
     }
 
     plugins.withId('java') {
-        sourceCompatibility = targetCompatibility = "1.8"
+        sourceCompatibility = targetCompatibility = JavaVersion.VERSION_11
```

Comment:
From description
> This is needed for 2.0.0, since 2.0.0 is using Apache Lucene 9, which requires JDK 11 or newer

Should we bump to 2.0 to test if this works?

Reply:
^ Yes, definitely do. But you're fine doing it in another PR.